source/skins/simple/raw/TooltipSkin.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import classnames from 'classnames';
+import { pickDOMProps } from '../../../utils/props';
+import { themr } from 'react-css-themr';
+import { TOOLTIP } from '../identifiers';
+import Bubble from '../../../components/Bubble';
+import SimpleBubbleSkin from '../BubbleSkin';
+
+export default themr(TOOLTIP)((props) => (
+  <span
+    {...pickDOMProps(props)}
+    className={classnames([
+      props.className,
+      props.theme.root,
+    ])}
+  >
+    <Bubble
+      skin={<SimpleBubbleSkin />}
```

Comment:
I just figured that i forgot to request this in the other PR: if you nest components you have to use a factory method (see `source/skins/simple/raw/AutocompleteSkin.js` on how to do this). Otherwise you will make it impossible to override the theme of the nested components because they are pinned to the default theme!

Reply:
By the factory method, do you just mean using `source/skins/simple/BubbleSkin` instead of `source/skins/simple/raw/BubbleSkin`?

Reply:
No, the `TooltipSkin` and `OptionsSkin` should export factory methods which take the nested components as params instead of importing (hard reference) them directly:

E.g: 
```js
export const tooltipSkinFactory = (BubbleSkin) => (
  (props) => { … }
);
```

Just checkout [AutocompleteSkin](https://github.com/input-output-hk/react-polymorph/blob/master/source/skins/simple/raw/AutocompleteSkin.js) for a reference (it takes the `FormFieldSkin` as configuration … i just saw that it should actually also take the `Options` component as config param -> this needs to be fixed in a separate PR).

Reply:
Ah yes, I was using the `Options` from `SelectSkin` as an example... that also needs to be fixed by the sounds of it ;)